libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -197,6 +197,12 @@ bool AC_WPNav::get_wp_destination(Location& destination) const
     if (!AP::ahrs().get_origin(destination)) {
         return false;
     }
+    if (_terrain_alt==true) {
+        destination.change_alt_frame(Location::AltFrame::ABOVE_TERRAIN);
+    }
```

Comment:
can you please use "} else {" instead

Reply:
Definitely, Sir!!Thank You for pointing it out